gapis/api/vulkan/api/renderpass_framebuffer.api

```diff
@@ -210,39 +210,16 @@ vkCmdBeginRenderPassArgs {
 }
 
 sub void dovkCmdBeginRenderPass(ref!vkCmdBeginRenderPassArgs args) {
-  VK_ATTACHMENT_UNUSED := as!u32(0xFFFFFFFF)
   lastDrawInfo().Framebuffer = Framebuffers[args.Framebuffer]
   lastDrawInfo().LastSubpass = 0
   lastDrawInfo().RenderPass = RenderPasses[args.RenderPass]
   lastDrawInfo().InRenderPass = true
-  for _ , _ , v in lastDrawInfo().Framebuffer.ImageAttachments {
+  for _ , i , v in lastDrawInfo().Framebuffer.ImageAttachments {
```

Comment:
humm...

Reply:
The keys in `ImageAttachments` are contiguous in the range `(0 .. let(ImageAttachments))` (see `vkCreateFramebuffer`). I think I will change these cases to use `for i in (0 .. len(imageAttachments))`, if there are no objections.